client/sites-dashboard/components/sites-site-coming-soon.tsx

```diff
@@ -0,0 +1,45 @@
+import styled from '@emotion/styled';
+import { CSSProperties } from 'react';
+
+type Props = {
+	siteName?: string;
+	className?: string;
```

Comment:
I think this `className` can be removed?

Reply:
We aren't making of it currently. I typically pass it down so you can do `styled(SiteComingSoon)` later and not have any issues with styles not taking effect. But I will remove it to simplify the props.